templates/main.html

```diff
@@ -36,12 +36,13 @@
 				<nav class="hidden-xs pull-right text-right">
 					<ul>
 						<li>
-							<a class="dropdown-target" aria-haspopup="true" href="{{ url_for('reports') }}">
+							<a class="dropdown-target" aria-haspopup="true">
```

Comment:
Could we use a different element here rather than `<a>` if we're removing the link functionality from it? Eg maybe `<span>`.

Reply:
You are right. Not per se against `span`, but do you think `button` would be more semantic. That would also "solve" the comment above.

Reply:
It's a hard call. `button`s typically change the state of the application. Is the dropdown menu open/close changing states? Maybe? As long as we persist the `aria-haspopup` attribute I don't think it matters much what element we use. But I do think leaving it as `a` has some unintended side-effects, like the underline on hover, and is even less semantic than span/button. I'll leave it up to you :)